### PR TITLE
Add episode 109

### PR DESCRIPTION
**Pull request by rumyantseva**

### CODE REVIEW

File: content/episode/2017/09-13-109.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+link = "golangshow.com/cdn/episodes/109.mp3"
+size = 17058725
+length = "00:40:46"
+categories = ["новости"]
++++
+
+## Темы
+
+- [Традиционный отчет от Саши](https://github.com/LK4D4/report/blob/master/reports/golang-09-13.md)
+  -  [testing: parallelize tests over count](https://github.com/golang/go/commit/f04d5836181dec3ec1b7e427607f02fa7a204a2d)
```
Comment (AlekSi) [inline content/episode/2017/09-13-109.md]: Two spaces.

File: content/episode/2017/09-13-109.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  -  [strconv: optimize Atoi for common case](https://github.com/golang/go/commit/46aa9f5437b000fad3696b0cd9fd97995da16411)
+  -  [fmt: document verbs %b %d %o %x %X for printing pointers](https://github.com/golang/go/commit/a4140b745ce22c56821750001f30fca4020b4650)
+  - [cmd/fix: rewrite x/net/context by default](https://github.com/golang/go/commit/bf90da97c1aaec78d2f8ad8b74a506d3b6f0ee75)
+  -  [math/rand: add Shuffle](https://github.com/golang/go/commit/a2dfe5d278eae0864397a046a8206342a426d2bd)
```
Comment (AlekSi) [inline content/episode/2017/09-13-109.md]: Two spaces.

File: content/episode/2017/09-13-109.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Проекты, библиотеки и примеры кода
+
+- https://github.com/goby-lang/goby
+- https://github.com/posener/complete
+- https://transform.now.sh/json-to-go/
+- https://gnorm.org
+
+## Конференции и митапы
+
+* [DevFest Siberia 2017](https://gdg-siberia.com) в Новосибирске, 23-24 сентября
```
Comment (AlekSi) [inline content/episode/2017/09-13-109.md]: Let's not mix `*` and `-` at least in the single document. I think we used `*` consistently before.

**Review by AlekSi — CHANGES_REQUESTED** [review]

File: content/episode/2017/09-13-109.md (hunk 3 above)
Comment (rumyantseva) [reply content/episode/2017/09-13-109.md]: Well, I always use `-` :) `*` is a copy-pasted part from the previous episode.
But I don't mind to use `*`, yeah.

Comment (rumyantseva) [reply content/episode/2017/09-13-109.md]: Fixed!

**Review by AlekSi — APPROVED** [review]

**Pull request merged**